www/js/sections.js

```diff
@@ -113,7 +125,7 @@ const processResponseStatus = response => {
 const extractResponseJSON = response => response.json()
 
 const fragmentForSection = section => {
-  const fragment = document.createDocumentFragment()
+  const fragment = lazyDocument.createDocumentFragment()
```

Comment:
I've tried updating the demos to use a DocumentFragment instead of a div but they seem to work fine. Since DocumentFragments avoid reflows, it would be a good debugging test to see if using a div changes the lazy load image reflow behavior.

Reply:
I'll check thanks!

Reply:
Tested with div and saw same behavior.